migrationtools/migrations.py

```diff
@@ -122,6 +122,8 @@ def migrate_registrations(self, start=None, stop=None, limit=None):
                             current_details=current_details, lang_code=lang, consent=consent,
                             sa_id_no=registration[reg_cols.mom_id_no],
                             mom_dob=registration[reg_cols.mom_dob], source=source, last_mc_reg_on=source)
+                    self.seed_identity.update_identity(identity[ident_cols.id], current_details,
+                                                       updated_at=registration[reg_cols.updated_at])
```

Comment:
These in place updates always leave me uncomfortable because they're generally easy places for bugs to hide in but if you're happy with it then cool.